vgstation13.dme

```diff
@@ -2178,7 +2156,6 @@
 #include "code\WorkInProgress\buildmode.dm"
 #include "code\WorkInProgress\explosion_particles.dm"
 #include "code\WorkInProgress\periodic_news.dm"
-#include "code\WorkInProgress\Cael_Aislinn\sculpture.dm"
```

Comment:
not sure why this and mine_vending were changed, any ideas?

Reply:
I dunno man it seems like the PR author would be the one to ask about that

Reply:
I'll ask him and see what he says.

Reply:
mine_vending isn't a thing anymore on Bleeding-Edge

Reply:
That'll get resolved when this gets merged onto bleeding-edge anyway